internal/bits/order_test.go

```diff
@@ -349,7 +349,8 @@ func TestOrderOfBytes(t *testing.T) {
 	err := quickCheck(func(values [][16]byte) bool {
 		slices := make([][]byte, len(values))
 		for i, v := range values {
-			slices[i] = v[:]
+			cpy := v
+			slices[i] = cpy[:]
```

Comment:
Why did we need the copy here?

Reply:
This was causing each item in `slices` to be a reference to the last item in `values`. Assigning it to an intermediate var was one way to resolve this.

Reply:
Ah interesting, the range variables are always full of surprises.

How about this version so we don't need the copy?
```go
for i := range values {
    slices[i] = values[i][:]
}
```

Reply:
Changed in f55e598